python/lsst/ctrl/oods/fileIngester.py

```diff
@@ -62,4 +62,7 @@ def run_task(self):
                 logger.info(f"{filename} ingested successfully.")
             except Exception as e:
                 logger.warning(f"{e}, moving to {self.bad_file_dir}.")
-                shutil.move(filename, self.bad_file_dir)
+                try:
+                    shutil.move(filename, self.bad_file_dir)
+                except OSError as ex:
+                    logger.error(f"Operation failed: {ex}.")
```

Comment:
Not having looked at this code for a while, my first thought was to wonder why the error wasn't thrown again for something later to catch.  A short comment that you did that on purpose because...  would be helpful.